Makefile

```diff
@@ -17,7 +17,10 @@ MERGE=true
 
 JAVAC=$(JAVA_HOME)/bin/javac
 JAR=$(JAVA_HOME)/bin/jar
-JAVAC_OPTIONS=-source 7 -target 7 -Xlint:-options
+ifeq ($(JAVAC_RELEASE_VERSION),)
+  export JAVAC_RELEASE_VERSION=7
+endif
```

Comment:
`ifeq` is redundant: command line arguments will override the value anyway.
I'd also prefer a shorter name (easier to type), e.g., `JAVAC_TARGET`.

Reply:
You're right, fixed it.